@olamy new error, trying to fix this one too.

```
[INFO] --- tycho-p2-repository-plugin:3.0.0:assemble-maven-repository (maven-p2-site) @ jetty-p2 ---
[ERROR] [93b6413b-c6ef-4bb1-b5ad-f5f04ab47384][plugin>org.eclipse.tycho:tycho-p2-repository-plugin:3.0.0] org.eclipse.tycho.p2.tools.impl
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.tycho.p2.tools.impl [41]
  Unresolved requirement: Require-Bundle: org.eclipse.equinox.p2.director.app
    -> Bundle-SymbolicName: org.eclipse.equinox.p2.director.app; bundle-version="1.2.200.v20220802-0626"; tags:List<String>="osgi.connect"; singleton:="true"
       org.eclipse.equinox.p2.director.app [53]
         Unresolved requirement: Import-Package: org.eclipse.equinox.p2.core; version="[2.8.0,3.0.0)"
```

--- a/jetty-p2/pom.xml
+++ b/jetty-p2/pom.xml
@@ -24,7 +24,7 @@
           <dependency>
             <groupId>org.eclipse.platform</groupId>
             <artifactId>org.eclipse.equinox.p2.core</artifactId>
-            <version>2.6.200</version>
+            <version>2.9.200</version>
           </dependency>
         </dependencies>
         <executions>

--- a/jetty-p2/pom.xml
+++ b/jetty-p2/pom.xml
@@ -12,7 +12,7 @@
   <description>Generates a (maven based) P2 Updatesite</description>
   <packaging>pom</packaging>
   <properties>
-    <tycho-version>3.0.0</tycho-version>
+    <tycho-version>2.7.5</tycho-version>
   </properties>
   <build>
     <plugins>
@@ -20,13 +20,6 @@
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-p2-repository-plugin</artifactId>
         <version>${tycho-version}</version>
-        <dependencies>
-          <dependency>
-            <groupId>org.eclipse.platform</groupId>
-            <artifactId>org.eclipse.equinox.p2.core</artifactId>
-            <version>2.9.200</version>
-          </dependency>
-        </dependencies>
         <executions>
           <execution>
             <configuration>

